github_issue_summarization/02_distributed_training.md

```diff
@@ -1,23 +1,26 @@
 # Distributed training using Estimator
 
-Distributed training with keras currently doesn't work; see
+Distributed training with Keras currently does not work. Do not follow this guide
```

Comment:
Perhaps in a follow on PR we should delete the code related to using TF.estimator? 
No one seems to be interested in fixing it. We can rely on git history to provide a version of the old code if anyone is interested in picking it up.